abdulfataiaka/pluralsight-admin
# feat(appUI): complete tutorial for app

#### What does this PR do
- Adds user interface and its functionalities for application

#### Description of work done
- Follow all modules to complete basic app requirements
- Fix failing test

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to the branch `ft-add-app-ui-161178456`
- Run `npm run app:init`
- Open the url `http://localhost:4000` on your browser
- Ensure all tests are passing by running `npm test`

#### Relevant pivotal tracker story
[161178456](https://www.pivotaltracker.com/story/show/161178456)

#### Screenshots
<img width="1440" alt="screen shot 2018-10-17 at 2 14 10 pm" src="https://user-images.githubusercontent.com/32123313/47088681-fb2ddb00-d216-11e8-85bd-d89313514b9f.png">
<img width="1440" alt="screen shot 2018-10-17 at 2 14 14 pm" src="https://user-images.githubusercontent.com/32123313/47088682-fb2ddb00-d216-11e8-8303-1f6a3bdadbb5.png">
<img width="1440" alt="screen shot 2018-10-17 at 2 14 12 pm" src="https://user-images.githubusercontent.com/32123313/47088684-fb2ddb00-d216-11e8-97a3-6099debd091f.png">
<img width="1440" alt="screen shot 2018-10-17 at 2 14 20 pm" src="https://user-images.githubusercontent.com/32123313/47088686-fbc67180-d216-11e8-9b15-cf34ef071392.png">
<img width="1440" alt="screen shot 2018-10-17 at 2 14 16 pm" src="https://user-images.githubusercontent.com/32123313/47088687-fbc67180-d216-11e8-8ac5-5d262889b4f9.png">
